backstop/assets/multiSelectTag.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Web component for multi select tag. Use in other components examples to simplify html.
+ * <multi-select-tag value="Option 1"></multi-select-tag>
+ */
+class MultiSelectTag extends HTMLElement {
+  constructor() {
+    super();
+  }
+
+  connectedCallback() {
+    const value = this.getAttribute('value');
+    const dismissible = this.hasAttribute('dismissible');
+
+    const innerHtml = `
+    <span
+      class="iui-multi-select-tag"
+      tabindex="0"
+    >
+      <span
+        class="iui-multi-select-tag-label"
+        title="${value}"
+      >
+        ${value}
+      </span>
+      ${
+        dismissible
```

Comment:
Would it make sense to switch from nondismissable being the default to dismissable being the default? Not sure what the expected default would be from a user's viewpoint.

Reply:
Maybe it would, but I didnt like the prop name in that case, so left as this (or couldnt figure one). I personally prefer to add features through props than to disabling them. In react we add `close` button only if user provides `onRemove` callback, to answer your question about the default.
Maybe it was not clear, but these web components in css repo are just to make our lives easier. Readability is better and we do not need to flood the same html. So thats the only purpose of this, it is not exposed to the users in anyway and it is not something we will need to support actively.